### DISCUSSION TITLE
Which Firmware

### DISCUSSION THREAD

**Discussion by albertusQuelle** [a1ex4/ownfoil#81] [Q&A]
Hi, my Switch is unmodded. 
Which firmware do i need to use ownfoil? And can you also give me a good tutorial pls?

**Comment by Sev7nrayne** [comment]
1. You need a modded switch to install Tinfoil- https://tinfoil.io/Download 
2. Ownfoil is not dependent on Switch firmware.
3. Tutorial: Start here -> https://github.com/a1ex4/ownfoil?tab=readme-ov-file#docker-run
